netbox/extras/migrations/0129_fix_script_paths.py

```diff
@@ -0,0 +1,40 @@
+from django.conf import settings
+from django.core.files.storage import storages
+from django.db import migrations
+from urllib.parse import urlparse
+
+from extras.storage import ScriptFileSystemStorage
+
+
+def normalize(url):
+    parsed_url = urlparse(url)
+    if not parsed_url.path.endswith('/'):
+        return url + '/'
+    return url
+
+
+def fix_script_paths(apps, schema_editor):
+    """
+    Fix script paths for scripts that had incorrect path from NB 4.3.
+    """
+    storage = storages.create_storage(storages.backends["scripts"])
+    if not isinstance(storage, ScriptFileSystemStorage):
+        return
+
+    ScriptModule = apps.get_model('extras', 'ScriptModule')
+    script_root_path = normalize(settings.SCRIPTS_ROOT)
+    for script in ScriptModule.objects.all():
+        if script.file_path.startswith(script_root_path):
```

Comment:
I think you could simplify this by moving the 'startswith' bit in to the query.

```python
for script in ScriptModule.object.filter(file_path__startswith=script_root_path):
   script.file_path = ...
   script.save()
```

You might even be able to combine the update in to single SQL update, but that might not be worth the hassle.